### ISSUE TITLE
Links have changed

### ISSUE THREAD

**Issue by sicablondu** [dexterke/plugin.video.digi-online#6]
Hello,

I think the website links have changed. I modified the login link and one of the stations link but i am getting errors. Can you please assist to make this addon great again?

Thanks,
Marius

**Comment by dexterke** [comment]
Yes, just noticed today, www.digionline.ro/ completely changed, spanking new look, different website. I'll have a look, but this may take a while.

**Comment by dexterke** [comment]
plugin.video.digi-online is broken, github repo https://github.com/dexterke/plugin.video.digi-online is going away.

www.digi-online.ro is now www.digionline.ro, new website uses TLS/SSL encryption and a different playback mechanism, I have written a new plugin to replace this one, it's called "Digi.ro-Online" https://github.com/dexterke/plugin.video.digi.ro-online

Please start testing the new plugin, your feedback would be highly appreciated.
